### PR TITLE
Prevent use of internal API's to hinder downtime of 4insight

### PR DESCRIPTION
**Pull request by hanne-opseth-rygg-4ss**
## Description
To regulate python's use of internal API's to prevent downtime of 4insight. For now, it is not included in the user guide. This is coordinated with Nikolai and Heidi. 

## Checklist
- [ ] PR title is descriptive and fit for injection into release notes (see tips below)
- [ ] Correct label(s) are used

PR title tips:
* Use imperative mood
* Describe the motivation for change, issue that has been solved or what has been improved - not how
* Examples:
  * Add functionality for Allan variance to sensor_4s.simulate
  * Upgrade to support Python 9.10
  * Remove MacOS from CI
  

[ESS-2426]: https://4insight.atlassian.net/browse/ESS-2426?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, macos-latest, windows-latest]
-        python-version: [3.8, 3.9, "3.10"]
+        python-version: [3.8, 3.9]
```
Comment (hanne-opseth-rygg-4ss) [inline .github/workflows/ci.yaml:16]: The tests crashed when using python version 3.10. This is not related to the code I have implemented, therefore I only removed this python version. However, I think this should be solved, but maybe that's part of another PR?

Comment (branislav-jenco-4ss) [reply .github/workflows/ci.yaml]: I wanted to see what the crash was about with 3.10 so pushed a change where I added it back. It looks like the tests pass anyway, I think previous fails were just because the other jobs failed :)

Comment (hanne-opseth-rygg-4ss) [reply .github/workflows/ci.yaml]: okay, but when I run the tests locally I get that the same one fails(?), do you experience the same?
But will have a look on this in another PR

**Comment by hanne-opseth-rygg-4ss** [comment]
Yes, maybe add a section in the confluence support-page (https://4insight.atlassian.net/wiki/spaces/4/pages/162562049/Common+Support+requests) 

But what should be the way we allow it? Downgrade to an earlier version of the package or directly comment out the function after they have installed the package (given that they run it locally)?

**Comment by hanne-opseth-rygg-4ss** [comment]
@branislav-jenco-4ss 
Wrote in our common support requests-confluence how to handle it when users ask for allowance to use our internal-api. If everything looks okay, can you approve? :)

**Review by branislav-jenco-4ss — APPROVED** [review]

**Comment by branislav-jenco-4ss** [comment]
Woops, I thought I did!
